File: pdlearning/src/web-apps/src/libraries/domain-api/src/calendar/models/team-member-event.model.ts
export interface ITeamMemberEventModel {
  id: string;
  title: string;
  subTitle: string;
  startAt: Date;
  endAt: Date;
  parentId: string;
}

export class TeamMemberEventModel implements ITeamMemberEventModel {
  public id: string;
  public title: string;
  public subTitle: string;
  public startAt: Date;
  public endAt: Date;
  public parentId: string;

  constructor(data: ITeamMemberEventModel) {
    this.id = data.id;
    this.title = data.title;
    this.subTitle = data.subTitle;
    this.startAt = data.startAt;
    this.endAt = data.endAt;
    this.parentId = data.parentId;
  }
}

File: pdlearning/src/microservices/calendar-microservice/Api/Application/Models/TeamMemberEventModel.cs
using System;

namespace Microservice.Calendar.Application.Models
{
    public class TeamMemberEventModel
    {
        public Guid Id { get; set; }

        public Guid ParentId { get; set; }

        public string Title { get; set; }

        public string SubTitle { get; set; }

        public DateTime StartAt { get; set; }

        public DateTime EndAt { get; set; }
    }
}
